core/src/main/scala/Json.scala

```diff
@@ -602,7 +599,7 @@ object Json {
   implicit lazy val RuntimeSummaryEncoder: EncodeJson[Nelson.RuntimeSummary] =
     EncodeJson((rs: Nelson.RuntimeSummary) =>
       ("scheduler" := rs.deployment) ->:
-      ("consul_health" := rs.health) ->:
+      ("healthy" := rs.healthy) ->:
```

Comment:
This would be a breaking API change - if we are OK with this we would need to update the Nelson CLI: https://github.com/getnelson/nelson-cli/blob/a0ac4436fd853db60f74d565db446459a31ad2cb/src/github.com/getnelson/nelson/stack.go#L401